### DISCUSSION TITLE
Want to Migrate repository from bitbucket to GitHub

### DISCUSSION THREAD

**Discussion by santoshaurus** [community/community#54822] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I am using the following steps to relocate the repo.

```
git clone https://abhi_vyrill@bitbucket.org/kiran_shirapure/vyrill_widgets.git --mirror

cd shopify

git remote add upstream https://github.com/Vyrill/shopify.git

git push upstream master

git push –-tags upstream
```
 I do the last command (`git push --tags upstream`), I am having the following problem
 
 ```
 skumbhar@skumbhar produkme.git]$ git push --tags upstream
Username for 'https://github.com': santoshaurus
Password for 'https://santoshaurus@github.com': 
Enumerating objects: 278589, done.
Counting objects: 100% (278464/278464), done.
Delta compression using up to 4 threads
Compressing objects: 100% (41763/41763), done.
Writing objects: 100% (278331/278331), 889.54 MiB | 14.28 MiB/s, done.
Total 278331 (delta 239462), reused 271735 (delta 233686), pack-reused 0
remote: Resolving deltas: 100% (239462/239462), completed with 73 local objects.
remote: warning: File lightbulb/db_dump/bm_demo_6Sep2017/vyrill_staging/searches.bson is 64.15 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: error: Trace: f9d175d65bade1d1323d0f986190ebbb93db23635b217417dd704299c8a448b4
remote: error: See https://gh.io/lfs for more information.
remote: error: File lightbulb/db_dump/bm_demo_6Sep2017/vyrill_staging/videos.bson is 127.64 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File lightbulb/db_dump/bm_demo_6Sep2017/vyrill_staging/youtube_video_apis.bson is 407.55 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.
To https://github.com/Vyrill/back-end.git
 ! [remote rejected]       v1.0.0 -> v1.0.0 (pre-receive hook declined)
error: failed to push some refs to 'https://github.com/Vyrill/back-end.git'
```

When I investigated that problem, I found that the former back-end developer had uploaded a database dump to the current repository. That database contains a lot of files that are bigger than 100 MB. But GitHub blocks files larger than 100 MB. So we need to remove those big files before pushing this repository to GitHub.

I need help how to delete that big files in the bitbucket history 

Thank You

**Comment by airtower-luna** [comment, marked as answer]
See here: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository The process for too large files is the same, you can ignore the parts about making sure they're gone from all caches too.

The generally recommended (by the Git docs) approach for bulk editing history is [`git-filter-repo`](https://github.com/newren/git-filter-repo), the second tool described in that documentation.

Mind that editing history will change *all* commit IDs after the first edited commit. :warning:

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @santoshaurus! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
